Mathlib/Topology/PartialHomeomorph.lean

```diff
@@ -673,14 +686,12 @@ theorem isOpen_inter_preimage_symm {s : Set α} (hs : IsOpen s) : IsOpen (e.targ
 /-- The image of an open set in the source is open. -/
 theorem image_isOpen_of_isOpen {s : Set α} (hs : IsOpen s) (h : s ⊆ e.source) :
```

Comment:
We should just excise this lemma since it is an exact duplicate of `PartialHomeomorph.isOpen_image_of_subset_source`.

Reply:
Sure: except that this lemma was first. Which name do you prefer?

Reply:
I prefer `PartialHomeomorph.isOpen_image_of_subset_source` since it is available earlier and is correctly named.